kbase-extension/scss/partials/_narrative.scss

```diff
@@ -603,7 +611,7 @@ nav[role=navigation] ul {
   }
 }
 
-input#search_terms {
+input#search_terms { /* FIXME */
```

Comment:
Ideally you shouldn't need to have both an element and an ID declared. I couldn't find this element, or the following `#searchspan`, in the UI or the JS to check whether or not the element was necessary.

Reply:
if it's cruft should we jettison it?